config/crds/cluster.k8s.io_machinesets.yaml

```diff
@@ -18,7 +18,6 @@ spec:
       labelSelectorPath: .status.labelSelector
       specReplicasPath: .spec.replicas
       statusReplicasPath: .status.replicas
-    status: {}
```

Comment:
What happened to the status subresource?

Reply:
Same as above